spec/mspec/lib/mspec/matchers/include_any_of.rb

```diff
@@ -0,0 +1,31 @@
+class IncludeAnyOfMatcher
+  def initialize(*expected)
+    @expected = expected
+  end
+
+  def matches?(actual)
+    @actual = actual
+    @matched = []
+    @expected.each do |e|
+      if @actual.include?(e)
+        @matched.push e
+      end
+    end
+    return !@matched.empty?
+  end
+
+  def failure_message
+    ["Expected #{@actual.inspect}", "to include any of #{@expected.inspect}"]
+  end
+
+  def negative_failure_message
+    ["Expected #{@actual.inspect}", "not to include any of #{@matched.inspect}"]
```

Comment:
I think it would simpler to just use `@expected.inspect` like in `failure_message`, and just use a boolean in `matches?`.

Reply:
The idea is that this gives you a list of things which matched which you didn't want to match, rather than the whole list of things which didn't match.

Reply:
Right, but I think showing actual and expected, unchanged, is more similar to what other matchers do and so less surprising in general.
Also, the tests don't cover a case where `@matched` is different than `@expected`.

Reply:
Fixing.